demo/scripts/k8s_push_start.sh

```diff
@@ -2,6 +2,7 @@
 
 set -x
 
+dir=$(dirname "$0")
```

Comment:
Doesn't really matter, but maybe it should be `DIR` to be consistent with the rest?

Reply:
I made it consistent with the rest of the variables in this file. They're all lowercase for some reason

Reply:
OK